lib/tools.js

```diff
@@ -2733,6 +2733,54 @@ function isLocalStatesDbServer(dbType, host, checkIfLocalOnly) {
     return result;
 }
 
+/**
+ * Get ordered instances according to tier level
+ *
+ * @param {object} objects - Objects DB
+ * @param {object} logger - logger object
+ * @param {string} [logPrefix] - prefix for logging
+ * @return {Promise<object[]>}
+ */
+async function getOrderedInstances(objects, logger, logPrefix='') {
```

Comment:
add ordering info in method name? I know "gteInstanesOrderedByStartPriority" is lengthy bt more clear